projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -461,6 +469,7 @@ export class OwcJsonService {
       return this.createTmsLayerFromOffering(offering, resource, context, targetProjection);
     } else {
       console.error(`This type of service (${layerType}) has not been implemented yet.`);
```

Comment:
I think here it would be good if you also log the resource or offering so that the use knows what is causing this problem